sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/GenderValuesDE.java

```diff
@@ -0,0 +1,23 @@
+package org.sormas.e2etests.enums;
+
+import java.util.Random;
+import lombok.Getter;
+
+@Getter
+public enum GenderValuesDE {
+  MALE("M\u00E4nnlich"),
+  FEMALE("Weiblich"),
+  OTHER("Divers"),
+  UNKNOWN("Unbekannt");
+
+  private final String gender;
+
+  GenderValuesDE(String humanGender) {
+    gender = humanGender;
+  }
+
+  public static String getRandomGenderDE() {
+    Random random = new Random();
+    return String.valueOf(GenderValuesDE.values()[random.nextInt(values().length)].gender);
+  }
+}
```

Comment:
help me out a bit to understand what's the need of this new class because UI text is displayed in english, and I can't see in the steps to change the language to german so we can validate the language differences.
My suggestion would be to have this data in the existing GenderValues, and we can refactor the class to store the translation for each locale, and create methods where we pass the desired locale and get the correct translation